tardis/__about__.py

```diff
@@ -3,8 +3,8 @@
 __summary__ = "Transparent Adaptive Resource Dynamic Integration System"
 __url__ = "https://github.com/matterminers/tardis"
 
-__version__ = "0.3.0"
+__version__ = "0.4.0"
 __author__ = "Manuel Giffels, Matthias Schnepf"
 __email__ = "giffels@gmail.com"
-__copyright__ = "2019 %s" % __author__
+__copyright__ = "2020 %s and Contributors" % __author__
```

Comment:
We're youngsters, but not toddlers! 🥳 

```suggestion
__copyright__ = "2018 - 2020 %s and Contributors" % __author__
```